test/jpypetest/test.py

```diff
@@ -0,0 +1,3 @@
+import platform
```

Comment:
delete me :)

Reply:
Ouch sorry.  I don't know how those two found their way into the commit.